docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,222 @@
 ---
 title: "Deploy to Kubernetes"
 sidebar_position: 21
----
\ No newline at end of file
+---
```

Comment:
todo - find a good place to put the links to our helm repo and docker image registry